Story/OnlineUIComponents/OnlineHud.cs

```diff
@@ -48,6 +48,8 @@ public void PlayerRemoved(ClientSettings clientSettings)
 
         public override void Update()
         {
+            friendsListToggled = Input.GetKeyDown(RainMeadow.rainMeadowOptions.FriendsListKey.Value) ^ (friendsListToggled && RainMeadow.rainMeadowOptions.FriendViewClickToActivate.Value);
```

Comment:
basically

```C
if (!toggle)
   show = keypressed;
if (toggle && keypressed)
   show = !show;
```

we'll never have to touch this again so there's no harm in making it clever

Reply:
oaugh forgot that [GetKeyDown is inconsistent](https://discord.com/channels/1094716194180841602/1094731805279658004/1286039559762874409)

gotta switch this to GetKey which is good actually